README.md

```diff
@@ -8,6 +8,10 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/KohlsTechnology/eunomia)](https://goreportcard.com/report/github.com/KohlsTechnology/eunomia)
 [![codecov](https://codecov.io/gh/KohlsTechnology/eunomia/branch/master/graph/badge.svg)](https://codecov.io/gh/KohlsTechnology/eunomia)
 
+## Getting started
```

Comment:
In "Template Engine" section there is no info on where CA_BUNDLE and SERVICE_CA_BUNDLE should come from

Reply:
They are being set in the [discoverEnvironment.sh](https://github.com/KohlsTechnology/eunomia/blob/master/template-processors/base/bin/discoverEnvironment.sh#L30-L34) script depending on the target environment (k8s or openshift).

It's discussed here:
- https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod
- https://docs.openshift.com/container-platform/3.10/dev_guide/secrets.html#service-serving-certificate-secrets

I'll link it